features/step_definitions/verification_process_steps.rb

```diff
@@ -103,6 +103,7 @@ def user(*traits)
 end
 
 Then(/^Individual should see cost saving documents for evidences$/) do
+  expect(page).to have_content('Income')
```

Comment:
Please use page object model here and use element selector rather than hard-coded english word

Reply:
@markgoho I can use POM here.

I want to make sure that there's specific content present on this page. I'm not sure how to verify the content with element selector unless the selector is unique.

Reply:
whatever container holds the word Income can be made unique with a class, id, or `data-` attribute

then, use an appropriate css selector to target the element